swiftsimio/visualisation/power_spectrum.py

```diff
@@ -406,7 +403,7 @@ def folded_depositions_to_power_spectrum(
                 maximally_sampled_wavenumber = np.max(
                     folded_wavenumber_centers[use_bins]
                 )
-                cutoff_wavenumber = min(
+                cutoff_wavenumber = np.min(
                     cutoff_above_wavenumber_fraction * maximally_sampled_wavenumber,
                     cutoff_above_wavenumber_fraction * cutoff_wavenumber,
                 )
```

Comment:
Can you explain to me (not necessary to change the code) why this change? Does `min` not provide the appropriate comparison in a cosmo_quantity sense?

Reply:
Plain `min` completely bypasses the `__array_function__` method that will catch that we're dealing with numpy arrays or subclasses initially. It just compares the arguments with `__lt__`, or maybe `__gt__`, which might then get back into treating them as numpy-like. This is better for developer sanity.